pkg/pmem-ns-init/main.go

```diff
@@ -72,10 +72,12 @@ func initNVdimms(ctx *ndctl.Context, useforfsdax int, useforsector int) {
 }
 
 func createNS(r *ndctl.Region, uselimit int, nsmode ndctl.NamespaceMode) {
+	align := uint64(1024 * 1024 * 1024)
```

Comment:
Same as in the other commit: `align` here isn't really about alignment, it's a minimum size, right?

Reply:
align here is actually exactly for alignment, used down in
r.CreateNamespace call:
```
Align: align,
```

The confusion is created by my lazy approach to use same value for min.size. I will make it cleaner.